src/test/kotlin/no/nav/syfo/testutil/TestKafka.kt

```diff
@@ -60,9 +57,10 @@ fun testPersonoppgavehendelseProducer(
 fun testPersonoppgavehendelseConsumer(
     environment: Environment,
 ): KafkaConsumer<String, String> {
-    val consumerPropertiesPersonoppgavehendelse = kafkaConsumerConfig(env = environment)
+    val consumerPropertiesPersonoppgavehendelse = kafkaAivenConsumerConfig(environment.kafka)
         .overrideForTest()
         .apply {
+            put(ConsumerConfig.GROUP_ID_CONFIG, "ispersonoppgave-v1")
             put("specific.avro.reader", false)
```

Comment:
Trengs denne nå som avro er fjernet?

Reply:
Litt usikker på den der: avro brukes jo fortsatt på et vis for disse hendelsene (har feks fortsatt en dependency på implementation("no.nav.syfo.dialogmote.avro:isdialogmote-schema:${Versions.isdialogmoteSchema}"))